tests/src/JIT/SIMD/AbsGeneric.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using System.Collections.Generic;
+using System.Numerics;
+
+namespace VectorMathTests
+{
+    class Program
+    {
+
+
+
+        static int Main(string[] args)
```

Comment:
nit: some files have extra whitespace while other `.cs` tests files under `JIT/` don't have extra linefeeds between `class name \n{` and the first method.